tests/publiccloud/prepare_tools.pm

```diff
@@ -22,6 +22,13 @@ use registration 'add_suseconnect_product';
 use version_utils qw(is_sle is_opensuse);
 use repo_tools 'generate_version';
 
+sub wrap_executable {
```

Comment:
```suggestion
sub spurt_file {
```
or
```suggestion
sub write_file {
```
or 
```suggestion
sub create_script_file {
```

Reply:
1. not sure what  `spurt_file` mean :) 
2. `write_file` and `create_script_file` not fully describe what actually happen in function 
 
but  on the other hand `wrap_executable` also not the best one :) 
so I will randomly pick `create_script_file` :P